{{cookiecutter.project_slug}}/.github/dependabot.yml

```diff
@@ -43,6 +51,10 @@ updates:
     # Every weekday
     schedule:
       interval: 'daily'
+    groups:
+      docker:
+        patterns:
+          - '*'
```

Comment:
I'm not sure about this one, each docker image in this group are quite unrelated so I would personally prefer to keep them separated. They are not being updated very frequently, so grouping them isn't solving much IMO.

I think that could potentially bundle 2 updates that I would like to split: e.g a major node update with a patch traefik update... Major node would require more attention or work.